src/components/drive-container.tsx

```diff
@@ -1,160 +1,23 @@
-                                    name: 'switchboard-push',
-                                    transmitterType: 'SwitchboardPush',
-                                },
-                                filter: {
-                                    branch: ['main'],
-                                    documentId: ['*'],
-                                    documentType: ['*'],
-                                    scope: ['global'],
-                                },
-                                label: 'Switchboard Sync',
-                                listenerId: '1',
-                                system: true,
-                            },
-                        ],
-                        triggers: [],
-                        pullInterval: 3000,
-                    });
-                } else {
-                    await addDrive({
-                        global: {
-                            id: isRemoteDriveInput(input) ? input.id : '',
-                            icon: null,
-                            name: input.driveName,
-                            slug: null,
-                        },
-                        local: {
-                            availableOffline: input.availableOffline,
-                            sharingType: input.sharingType.toLowerCase(),
-                            listeners: isRemoteDriveInput(input)
-                                ? [
-                                      {
-                                          block: true,
-                                          callInfo: {
-                                              data: input.url,
-                                              name: 'switchboard-push',
-                                              transmitterType:
-                                                  'SwitchboardPush',
-                                          },
-                                          filter: {
-                                              branch: ['main'],
-                                              documentId: ['*'],
-                                              documentType: ['*'],
-                                              scope: ['global'],
-                                          },
-                                          label: 'Switchboard Sync',
-                                          listenerId: '1',
-                                          system: true,
-                                      },
-                                  ]
-                                : [],
-                            triggers: [],
-                        },
-                    });
-                }
-
-                toast(t('notifications.addDriveSuccess'), {
-                    type: 'connect-success',
-                });
-            } catch (e) {
-                console.error(e);
-            }
-        };
+export default function DriveContainer() {
+    const uiNodes = useUiNodes();
```

Comment:
business logic is moved to the `useUiNodes` hook, so this component only needs to worry about rendering.